src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -124,6 +133,7 @@ export default class CardanoSignTransaction extends AbstractMethod {
             metadata: payload.metadata,
             protocolMagic: payload.protocolMagic,
             networkId: payload.networkId,
+            validityIntervalStart: payload.validityIntervalStart,
```

Comment:
I'd move after metadata.

Reply:
I originally added it at the end on the premise that the outward facing API also has parameters but now that I double checked it actually takes an object, so this totally makes sense, I will fix it